Jenkinsfile

```diff
@@ -236,11 +233,10 @@ helpers.rootLinuxNode(env, {
                                     wrap([$class: 'Xvfb']) {
                                         println "Test Windows JS"
                                         dir("visdiff") {
-                                            bat "npm install"
+                                            bat "yarn"
                                         }
-                                        bat "npm install .\\visdiff"
                                         dir("desktop") {
-                                            bat "npm run vendor-install"
```

Comment:
:sheep: Will need to do something here for electron, different from vendor-install. I'll look into this.